### ISSUE TITLE
Show items that are ready for merge

### ISSUE THREAD

**Issue by tvansteenburgh** [juju-solutions/review-queue-classic#29]
I'd like to easily see which items have gotten a +1 but are not yet merged, so those items can be merged and removed from the queue.

**Comment by AdamIsrael** [comment]
This kind of fits with something I've been mulling over; a way to better prioritize the queue. 

There seem to be a few different use cases where a charm should get attention outside of the usual FIFO order. For example, if you're a charmer, the charms that need a charmers attention should be called out. If you belong to a specific team that has a charm up for review (like cloudfoundry or big data), then those charms would also be called out.

There's also the case of merge proposals that are very minor, i.e., icon or README changes, that could be easily approved and merged without having to wait a few weeks.

**Comment by whitmo** [comment]
maybe a simple tagging system could facilitate prioritization?

-w

On Fri, Mar 6, 2015 at 1:26 PM, Adam Israel notifications@github.com
wrote:

> This kind of fits with something I've been mulling over; a way to better
> prioritize the queue.
> 
> There seem to be a few different use cases where a charm should get
> attention outside of the usual FIFO order. For example, if you're a
> charmer, the charms that need a charmers attention should be called out. If
> you belong to a specific team that has a charm up for review (like
> cloudfoundry or big data), then those charms would also be called out.
> 
> There's also the case of merge proposals that are very minor, i.e., icon
> or README changes, that could be easily approved and merged without having
> to wait a few weeks.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/juju-solutions/review-queue/issues/29#issuecomment-77619671
> .

## 

<=>
david "whit" morriss

 "If you don't know where you are,
  you don't know anything at all"

  Dr. Edgar Spencer, Ph.D., 1995

**Comment by marcoceppi** [comment]
Yeah, some way to promote an item from needs review to needs a charmer. We could just use keywords like "LGTM" or we could add a button. Not sure which is the best way. Opinions?

**Comment by johnsca** [comment]
I think triggering off of review status, keyword, etc, is nice, but has been proven to be not 100% reliable.  So, I'd say we should definitely have a button as a fall-back at the very least, and thus it makes sense to do as the first implementation.

Also, the button could (at some point) have additional actions to manually change to other statuses as well (e.g., blocked on awaiting user response), in case the automatic detection gets it wrong.  I think that button would go a long way toward alleviating the issues we've been having with the Queue by giving us a direct way to correct guesses by the ingestion system.
